packages/shared/src/utils/currency/currency.js

```diff
@@ -82,64 +82,137 @@ export const CryptoConfig = (() => {
     let crypto_config;
 
     // TODO: [use-shared-i18n] - Use translate function shared among apps or pass in translated names externally.
-    const initCryptoConfig = () => ({
-        BTC: {
-            display_code: 'BTC',
-            name: 'Bitcoin',
-            min_withdrawal: 0.002,
-            pa_max_withdrawal: 5,
-            pa_min_withdrawal: 0.002,
-            fractional_digits: 8,
-        },
-        BCH: {
-            display_code: 'BCH',
-            name: 'Bitcoin Cash',
-            min_withdrawal: 0.002,
-            pa_max_withdrawal: 5,
-            pa_min_withdrawal: 0.002,
-            fractional_digits: 8,
-        },
-        ETH: {
-            display_code: 'ETH',
-            name: 'Ether',
-            min_withdrawal: 0.002,
-            pa_max_withdrawal: 5,
-            pa_min_withdrawal: 0.002,
-            fractional_digits: 8,
-        },
-        ETC: {
-            display_code: 'ETC',
-            name: 'Ether Classic',
-            min_withdrawal: 0.002,
-            pa_max_withdrawal: 5,
-            pa_min_withdrawal: 0.002,
-            fractional_digits: 8,
-        },
-        LTC: {
-            display_code: 'LTC',
-            name: 'Litecoin',
-            min_withdrawal: 0.002,
-            pa_max_withdrawal: 5,
-            pa_min_withdrawal: 0.002,
-            fractional_digits: 8,
-        },
-        UST: {
-            display_code: 'USDT',
-            name: 'Tether',
-            min_withdrawal: 0.02,
-            pa_max_withdrawal: 2000,
-            pa_min_withdrawal: 10,
-            fractional_digits: 2,
-        },
-        USB: {
-            display_code: 'USB',
-            name: 'Binary Coin',
-            min_withdrawal: 0.02,
-            pa_max_withdrawal: 2000,
-            pa_min_withdrawal: 10,
-            fractional_digits: 2,
-        },
-    });
+    const initCryptoConfig = () =>
+        Object.freeze({
```

Comment:
Freeze is only applied to the objects top level properties - so for the nested properties  so you could either use a `deepFreeze` function or `Object.freeze` each of them